spec/plotting/nyaplot/vector_spec.rb

```diff
@@ -37,7 +34,6 @@
     context type.to_s do
       it 'does not set x axis' do
         expect(plot).to receive(:add).with(type, [11, 22, 33]).ordered
-        expect(plot).to receive(:show).ordered
 
         vector.plot(type: type)
```

Comment:
I believe this should be enough:
```ruby
expect(vector.plot(type: type)).to eq plot
```
It exactly checks that `#plot` method would return an instance of plot constructed, or am I missing something?